src/response.rs

```diff
@@ -91,31 +56,6 @@ impl Response {
     /// Create a new response.
     #[cfg(not(feature = "unstable"))]
```

Comment:
is this line still needed?

Reply:
I don't think so because there isn't any unstable feature anymore. I will remove those. Thx!